content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -109,32 +134,12 @@ generic aws-secret \
 --from-file=creds=./aws-credentials.txt
 ```
 
-View the secret with `kubectl describe secret`
-
-{{< hint type="note" >}}
-The size may be larger if there are extra blank spaces in your text file.
-{{< /hint >}}
-
-```shell {copy-lines="1"}
-kubectl describe secret aws-secret -n crossplane-system
-Name:         aws-secret
-Namespace:    crossplane-system
-Labels:       <none>
-Annotations:  <none>
-
-Type:  Opaque
-
-Data
-====
-creds:  114 bytes
-```
-
 ## Create a ProviderConfig
 A {{< hover label="providerconfig" line="3">}}ProviderConfig{{</ hover >}}
-customizes the settings of the AWS Provider.  
+customizes the settings of the AWS Provider.
 
-Apply the 
-{{< hover label="providerconfig" line="3">}}ProviderConfig{{</ hover >}} 
+Apply the
+{{< hover label="providerconfig" line="3">}}ProviderConfig{{</ hover >}}
 with the this Kubernetes configuration file:
 ```yaml {label="providerconfig",copy-lines="all"}
 cat <<EOF | kubectl apply -f -
```

Comment:
I know this is subjective but per my rant about `cat <<EOF` in examples yesterday I've been omitting these here docs from the getting started with composition examples.

Instead I'm just saying (e.g.) "Save this as `xrd.yaml`" and showing the kubectl command to apply it.

Reply:
I think we should be consistent, so I'll follow what you're doing, but I have realized when running through this guide multiple times that I do actually like the single step copy/paste that heredocs enable. As opposed to copying the content, saving it to a file, then having to get a terminal to that location too, which is more steps 😇

Reply:
I made this update and when I ran through the guide to test, it definitely felt slower and more clumsy to have to save things to files and then apply them. I do prefer the zippy copy pasty heredocs! 😂

Reply:
Long term I'd like to go back to a variant of what we used to do:

* Add the YAML manifests as distinct files
* Embed the YAML file in the examples (to avoid duplicates)
* Provide a command that applies the file by URL - e.g. `kubectl apply -f https://docs.crossplane.io/...`

This is what Kubernetes does - e.g. https://kubernetes.io/docs/tutorials/configuration/updating-configuration-via-a-configmap/#rollout-configmap-volume

I find it to be a sweet spot. If you want to just run the example in your shell you can easily copy the command. If you want to copy the YAML and edit it in vim or YAML you don't have a weird here doc getting in the way.